opensanctions/crawlers/md_interdictie.py

```diff
@@ -77,27 +104,20 @@ def crawl(context: Context):
         delay_until_date = parse_delay(context, cells[IDX_DELAY_UNTIL].text_content().strip())
         start_date = delay_until_date or parse_date(cells[IDX_REGISTRATION_DATE].text_content().strip())
         
-        sanction_num = parse_sanction_num(context, cells[IDX_DECISION_NUM_DATE].text_content().strip())
+        sanction_num, decision_date = parse_sanction_decision(context, cells[IDX_DECISION_NUM_DATE].text_content().strip())
         sanction = h.make_sanction(context, entity, sanction_num)
         sanction.add("authorityId", sanction_num)
         sanction.add("reason", cells[IDX_REASON].text_content().strip(), lang="ro")
         sanction.add("startDate", start_date)
         sanction.add("endDate", parse_date(cells[IDX_END_DATE].text_content().strip()))
         sanction.add("listingDate", parse_date(cells[IDX_REGISTRATION_DATE].text_content().strip()))
 
-        parse_control(context, cells[IDX_ADMINS_FOUNDERS].text_content().strip())
-
-        # Persons
-
-
-        # Ownerships
-
-        # Memberships
-
-        
+        control_entities = parse_control(context, entity, decision_date, cells[IDX_ADMINS_FOUNDERS].text_content().strip())        
 
         context.emit(entity, target=True)
         context.emit(sanction)
+        for control_entity in control_entities:
```

Comment:
Why not emit straight in `parse_control`?

Reply:
Just to be a bit more functional. On the other hand there's a convention of crawl_ also emitting and make_ returning. In this case it seems to make sense to change this to crawl_control and emit there